src/Microsoft.TestPlatform.PlatformAbstractions/common/Tracing/PlatformEqtTrace.cs

```diff
@@ -131,6 +130,35 @@ public static string ErrorOnInitialization
             set;
         }
 
+        // This is added to ensure that traceSource should not be instantiated in when creating appdomains if EqtTrace is not enabled.
+        internal static bool DoNotInitialize
```

Comment:
No...
Default value of DoNotInitialize is `false`. This variable is only set while creating AppDomains. 
Since `PlatformEqtTrace` is not inheriting from `marshalbyrefobject` or Serializable, this value will not interfere with parent appdomain traces.